src/main/java/io/github/dlinov/leetcode/Task128LongestConsecutiveSequence.java

```diff
@@ -0,0 +1,30 @@
+package io.github.dlinov.leetcode;
+
+import java.util.*;
+
+public class Task128LongestConsecutiveSequence {
+    class Solution {
+        public int longestConsecutive(int[] nums) {
+        if (nums.length == 0) return 0;
+        final TreeSet<Integer> numSet = new TreeSet<>();
```

Comment:
That's a very novel idea for this problem that seems to work!

> Your algorithm should run in O(n) complexity.
I'd also suggest looking at the textbook [solution](https://github.com/linouk23/leetcode/blob/master/128.cpp) that runs in O(n).